templates/plugin.js

```diff
@@ -1,5 +1,5 @@
 import Vue from 'vue'
-import Vuetify from '<%= options.treeShake ? 'vuetify/lib' : 'vuetify' %>'
+import Vue from 'vuetify/lib'
```

Comment:
🤷‍♂️
```suggestion
import Vuetify from 'vuetify/lib'
```

Reply:
Seems you requested this while I was doing this change.